test-cases/tests/alerts/c14-verify-pvc-alerts.md

```diff
@@ -0,0 +1,14 @@
+---
+targets:
+  - 2.6.0
+---
+
+# C14 - Verify PVC alerts
+
+## Description
+
+More info: <https://issues.redhat.com/browse/INTLY-8413>
```

Comment:
@jhellar This should be https://issues.redhat.com/browse/INTLY-9432 (the 2.x one)